bongo/settings.php

```diff
@@ -30,7 +30,6 @@
  * @license     http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
 
-require_once(__DIR__ . '/../../../config.php');
```

Comment:
All references to config.php are not necessary unless the page is user-visible. As such, this one has been removed.